### ISSUE TITLE
Dust particle inclusion in macrophages

### ISSUE THREAD

**Issue by mgriese** [obophenotype/human-phenotype-ontology#5559]
Preferred term label:
Dust particle inclusion in macrophages

Synonyms

Definition (free text, please give PubMed ID)
Accumulation of inhaled, nondigestable particles in macrophages
Costabel U. Atlas of bronchoalveolar lavage. London: Chapman & Hall; 1998. ISBN0412792702.

Parent term (use hpo.jax.org/app)
Abnormalities of alveolar macrophages

Diseases characterized by this term ? (e.g. Orphanet or OMIM number)
ORPHA:182098 

Your nano-attribution (ORCID)
@GrieseAG
@katrinknoflach

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
Adding term as HP_0032985.

@GrieseAG
@katrinknoflach

can we provide a description of how the pathologist recognizes the dust accumulation?

**Comment by GrieseAG** [comment]
Comment: This can be recognized in May-Grünwald-Giemsa routine stains. Rarey and for special indications, BAL can be further analysed to identify and quantify  dust exposure. X-ray microprobe analysis identified particles which related to the known exposures, superimposed on a background of other particles related to smoking (kaolinite and mica) or to the general environment (silicon, titanium, and iron). Inhaled dust can follow in fibrous tissue and scarring which follow in impaired lung function.

**Comment by GrieseAG** [comment]
Reference: PMID: 3740951, Identification of inorganic dust particles in bronchoalveolar lavage macrophages by energy dispersive x-ray microanalysis. Johnson NF, Haslam PL, Dewar A, Newman-Taylor AJ, Turner-Warwick M.

**Comment by pnrobinson** [comment]
thanks! Added this additional information
